src/layers/Buffer.js

```diff
@@ -0,0 +1,49 @@
+import L from 'leaflet';
+import turfBuffer from '@turf/buffer';
+import { GeoJson } from './GeoJson';
+
+export const Buffer = GeoJson.extend({
+    options: {
+        radius: 1000,
+        style: {
+            color: '#95c8fb',
+            weight: 1,
+            pointerEvents: 'none',
+        },
+        opacityFactor: 0.2,
+        highlightStyle: false,
+    },
+
+    initialize(options = {}) {
+        // Replace polygons with buffered geometries
+        options.data = options.data.map(feature => 
+            feature.geometry.type !== 'Point' ? {
+                ...feature,
+                geometry: turfBuffer(feature, options.buffer / 1000).geometry, // km
+            } : feature
+        );
+
+        GeoJson.prototype.initialize.call(this, options);
+    },
+
+    pointToLayer(geojson, latlng) {
+        this.options.style.pane = this.options.pane;
```

Comment:
Should `this.options.pane` be initialized somewhere?  Maybe it's in the GeoJson parent class, checking

Reply:
Added defult pane value.